components/builder-api/src/server/services/metrics.rs

```diff
@@ -28,6 +28,7 @@ pub enum Counter {
     SingleUploadRequests,
     MultipartUploadRequests,
     DownloadRequests,
+    SizeRequests,
```

Comment:
I think we can remove the `SizeRequest` counter now because this will be part of `GetPackage`.